Makefile

```diff
@@ -96,6 +99,9 @@ release: ## Build release binaries
 		done ; \
 	done
 
+	$(KUBECTL) create -f config --dry-run=client -o yaml | \
+	  $(SED) 's|ko://github.com/triggermesh/triggermesh/cmd/\(.*\)|$(IMAGE_REPO)/\1:${IMAGE_TAG}|' > $(DIST_DIR)/triggermesh-core.yaml
```

Comment:
The name "core" was banned from our vocabulary for marketing reasons. We should avoid using it anywhere public.

Reply:
so just `triggermesh.yaml` then.